VoronoiLib/Structures/FortuneSite.cs

```diff
@@ -18,5 +19,73 @@ public FortuneSite(double x, double y)
             Cell = new List<VEdge>();
             Neighbors = new List<FortuneSite>();
         }
+
+        private List<VPoint> _points;
+        public List<VPoint> Points
+        {
+            get
+            {
+                if (_points == null)
+                {
+                    // it would probably be better to sort these as they are added to improve performance
+                    _points = new List<VPoint>();
+
+                    foreach (var edge in Cell)
+                    {
+                        _points.Add(edge.Start);
+                        _points.Add(edge.End);
```

Comment:
I believe this adds duplicates. After all, the `.Start` and `.End` of `VEdge`s are by definition shared with any "connecting" `VEdge`s in that `FortuneSite`. I am not sure if this is by design, but I don't think it is.

Reply:
This fork also appears to be dead, which is a shame, so I have stopped working on things related to this and have forgotten  most of the context so am not actively looking at this.